packages/opencensus-core/src/common/monitored-resource/types.ts

```diff
@@ -0,0 +1,74 @@
+export enum MonitoredResources {
+  GCP_GCE_INSTANCE = 'gce_instance',
```

Comment:
The type and attribute names should match [Specs](https://github.com/census-instrumentation/opencensus-specs/blob/master/resource/Resource.md). See also [go constants](https://github.com/census-ecosystem/opencensus-go-resource/blob/master/resourcekeys/const.go)